root/static/scripts/edit/URLCleanup.js

```diff
@@ -325,6 +322,18 @@ function findAmazonTld(url) {
   }
   return tld;
 }
+
+const linkToChannelMsg = l(
```

Comment:
This script is likely executed after the language is set, but better to not assume that's the case (or that we won't ever run this on the server, e.g. on seeded URLs once the edit forms are server-rendered by React).
```suggestion
const linkToChannelMsg = N_l(
```